include/pybind11/pybind11.h

```diff
@@ -543,7 +543,7 @@ class cpp_function : public function {
                         self_value_and_holder.type->dealloc(self_value_and_holder);
 
                     call.init_self = PyTuple_GET_ITEM(args_in, 0);
-                    call.args.push_back(reinterpret_cast<PyObject *>(&self_value_and_holder));
+                    call.args.emplace_back(reinterpret_cast<PyObject *>(&self_value_and_holder));
```

Comment:
In this case, it shouldn't matter, though? It's just a pointer...

Reply:
It either doesn't matter, or is better, so might as well consistently use the one that's better. :)

Reply:
Yeah, true, but ... poor `push_back`, being rendered useless :-(
Also, I do expect to see the constructor arguments in `emplace_back`, and somehow that doesn't fit well in my brain with pointers (I copy pointers, I don't copy-construct pointers :-D ).

But yes, this wasn't really a complaint.